### PR TITLE
Fix modulate/sim bug

### PR DESCRIPTION
**Pull request by firenoo**
Actionable.SIMULATE gives faulty results. Using Actionable.MODULATE will force the AE system to extract the correct amount, which fixes a dupe bug with the previous implementation.
Also close https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13805

### CODE REVIEW

File: src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java [hunk 1]
```diff
@@ -412,9 +410,7 @@ private void extractFluid(IAEFluidStack fluid, ItemStack fluidContainer, EntityP
         } else {
             partialStack.stackSize = 0;
         }
-        // Now perform the real extraction.
         actualExtract.setStackSize(toExtract);
-        this.host.getFluidInventory().extractItems(canExtract, Actionable.MODULATE, this.getActionSource());
```
Comment (miozune) [inline src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java:417]: This works as a fix for dupe, but also makes it harder to fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13805
Do you have any idea?

Comment (firenoo) [reply src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java:417]: I'm not familiar with how super tanks implement FluidContainer. How are locked fluids handled? I could try to insert 1mb of fake fluid before to see if it works and abort if it doesn't

Comment (miozune) [reply src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java:417]: It's not only super tank but in general `FluidContainer` might refuse to store fluid. You probably can use `IFluidContainerItem#drain` with `doDrain = false`

Comment (firenoo) [reply src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java:417]: Using FluidUtil now to check for fluid type in the container.

File: src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java [hunk 2]
```diff
@@ -346,23 +346,15 @@ private void extractFluid(IAEFluidStack fluid, ItemStack fluidContainer, EntityP
...
-        if (fluidContainer.getItem() instanceof IFluidContainerItem) {
-            capacity = ((IFluidContainerItem) fluidContainer.getItem()).getCapacity(fluidContainer);
-        } else if (FluidContainerRegistry.isContainer(fluidContainer)) {
-            // Fill with fluid container registry, or abort if it failed
-            ItemStack dummy = FluidContainerRegistry.fillFluidContainer(storedFluid.getFluidStack(), fluidContainer);
-            capacity = FluidContainerRegistry.getContainerCapacity(dummy);
-        } else {
-            return;
-        }
+        capacity = Util.FluidUtil.getCapacity(fluidContainer, storedFluid.getFluid());
```
Comment (miozune) [inline src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java:349]: nit: `int capacity =`

File: src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java [hunk 3]
```diff
@@ -412,9 +404,7 @@ private void extractFluid(IAEFluidStack fluid, ItemStack fluidContainer, EntityP
         } else {
             partialStack.stackSize = 0;
         }
-        // Now perform the real extraction.
         actualExtract.setStackSize(toExtract);
```
Comment (miozune) [inline src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java:407]: I think we can remove this?

**Pull request merged**